### ISSUE TITLE
Manpages installation is broken

### ISSUE THREAD

**Issue by spickartz** [eclipse/mosquitto#461]
Hello,

when I attempt to install the library via `make install`, I get

```
CMake Error at man/cmake_install.cmake:36 (file):
  file INSTALL cannot find
  "/home/pickartz/work/mosquitto/man/mosquitto_passwd.1".
```

It seems that the repo only contains the XML files but not the converted man-pages files.

Best regards,
Simon

**Comment by ralight** [comment]
Yes this is correct, the generated files are not in the repo. It should all build just fine if you are on Ubuntu - just run bare `make` first. If you aren't on Ubuntu then there are path problems, any solutions gratefully received.

**Comment by spickartz** [comment]
This does not seem to work for me, my calling sequence is:
```
$ cmake -DCMAKE_INSTALL_PREFIX=`pwd`/bin ..
$ make
$ make install
```

The last gives me:
```
CMake Error at man/cmake_install.cmake:36 (file):
  file INSTALL cannot find "/home/ubuntu/mosquitto/man/mosquitto_passwd.1".
Call Stack (most recent call first):
  cmake_install.cmake:49 (include)
```

I am running an Ubuntu 16.04.2 LTS (Xenial Xerus). Do you see what I am doing wrong here?

**Comment by ralight** [comment]
Ah, it's because the plain makefiles (no cmake) are the most supported option on anything apart from Windows and Mac (in other words, the cmake support is for Windows and Mac, but does mostly work on Linux/BSD).

The cmake scripts don't build the man pages, they just assume they are already there, which will be the case for the released versions. If you want to add support for building the man pages to the cmake scripts I'd take that as a pull request.

tldr; just use "make" instead of cmake first.
